bcda/cli.go

```diff
@@ -117,16 +121,16 @@ func importCCLF0(fileMetadata cclfFileMetadata) (map[string]cclfFileValidator, e
 	return validator, nil
 }
 
-func importCCLF8(fileMetadata cclfFileMetadata) error {
-	fmt.Printf("Importing CCLF8 file %s...\n", fileMetadata.name)
-	log.Infof("Importing CCLF8 file %s...", fileMetadata.name)
-
-	if (cclfFileMetadata{}) == fileMetadata {
+func importCCLF8(fileMetadata *cclfFileMetadata) error {
+	if fileMetadata == nil {
 		err := errors.New("file CCLF8 not found")
 		log.Error(err)
 		return err
 	}
 
+	fmt.Printf("Importing CCLF8 file %s...\n", fileMetadata.name)
```

Comment:
Could we have the stdout and log messages either before or after all the error handling, instead of in between?

Reply:
so the idea is to have it before the error handling, I remember we talked about that specifically in the code before. But we can't log fileMetadata.name if fileMetadata is null hence why it's after the null check.